internal/server/server.go

```diff
@@ -73,9 +84,10 @@ func WithListenAddress(addr string) Option {
 }
 
 // WithStreamBroker sets the event stream broker.
-func WithStreamBroker(broker events.Stream) Option {
+func WithStreamBroker(broker events.Stream, streamSubjectPrefix string) Option {
```

Comment:
As of now theres no intention on changing, this could be turned into a const in rivets going ahead so we have one less configuration param being passed around.